server/routes/one-time-code-auth/VerifyOneTimeCodeController.test.ts

```diff
@@ -64,7 +64,7 @@ describe('VerifyOneTimeCodeController', () => {
       )
 
       expect(req.flash).not.toHaveBeenCalled()
-      expect(res.redirect('/barcode/find-recipient'))
```

Comment:
All of these expectations on `res.redirect` were wrong (syntax), so weren't actually testing anything! Whoops! 😬